stack/rest/src/test/java/org/apache/usergrid/rest/management/RegistrationIT.java

```diff
@@ -17,212 +17,238 @@
+    public Map<String, Object> getRemoteTestProperties() {
+        return clientSetup.getRestClient().testPropertiesResource().get().getProperties();
+    }
 
-        try {
-            setTestProperty( PROPERTIES_SYSADMIN_APPROVES_ADMIN_USERS, "false" );
-            setTestProperty( PROPERTIES_SYSADMIN_APPROVES_ORGANIZATIONS, "false" );
-            setTestProperty( PROPERTIES_ADMIN_USERS_REQUIRE_CONFIRMATION, "false" );
-            setTestProperty( PROPERTIES_SYSADMIN_EMAIL, "sysadmin-1@mockserver.com" );
+    /**
+     * Sets a management service property locally and remotely.
+     */
+    public void setTestProperty(String key, Object value) {
+        // set the value remotely (in the Usergrid instance running in Tomcat classloader)
+        Entity props = new Entity();
+        props.put(key, value);
+        clientSetup.getRestClient().testPropertiesResource().post(props);
 
-//            JsonNode node = postCreateOrgAndAdmin( "test-org-1", "test-user-1", "Test User",
-//                    "test-user-1@mockserver.com", "testpassword" );
+    }
 
+    public void setTestProperties(Map<String, Object> props) {
+        Entity properties = new Entity();
+        // set the values locally (in the Usergrid instance here in the JUnit classloader
+        for (String key : props.keySet()) {
+            properties.put(key, props.get(key));
 
-            final String username = "registrationUser"+UUIDGenerator.newTimeUUID();
-            final String email = username+"@usergrid.com" ;
-            final String password = "password";
+        }
 
-            final TestUser user1 = new TestAppUser(username , password, email);
+        // set the values remotely (in the Usergrid instance running in Tomcat classloader)
+        clientSetup.getRestClient().testPropertiesResource().post(properties);
+    }
 
-            context.withOrg( "org" + UUIDGenerator.newTimeUUID() ).withApp( "app" + UUIDGenerator.newTimeUUID() ).withUser( user1 ).createNewOrgAndUser();
-            context.createAppForOrg();
+    public String getTokenFromMessage(Message msg) throws IOException, MessagingException {
+        String body = ((MimeMultipart) msg.getContent()).getBodyPart(0).getContent().toString();
+        // TODO better token extraction
+        // this is going to get the wrong string if the first part is not
+        // text/plain and the url isn't the last character in the email
+        return StringUtils.substringAfterLast(body, "token=");
+    }
 
-            final UUID owner_uuid = context.getActiveUser().getUuid();
+    public User postAddAdminToOrg(String organizationName, String email, String password, Token token) throws IOException {
+
+        ApiResponse apiResponse = this
+            .management()
+            .orgs()
+            .organization(organizationName)
+            .users()
+            .getResource(true, token)
+            .type(MediaType.APPLICATION_JSON_TYPE)
+            .accept(MediaType.APPLICATION_ATOM_XML_TYPE)
```

Comment:
What changed about this class such that we would need a different type or rather a different accept? Maybe a comment at the top of the class or for the tests that use this class?